src/Validation/PlValidation.php

```diff
@@ -1,33 +1,35 @@
 <?php
+declare(strict_types=1);
+
 /**
  * CakePHP(tm) : Rapid Development Framework (http://cakephp.org)
  * Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
  *
  * Licensed under The MIT License
  * Redistributions of files must retain the above copyright notice.
  *
- * @copyright     Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
- * @link          http://cakephp.org
- * @since         Localized Plugin v 0.1
- * @license       http://www.opensource.org/licenses/mit-license.php MIT License
+ * @copyright Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ * @link http://cakephp.org
+ * @since Localized Plugin v 0.1
+ * @license http://www.opensource.org/licenses/mit-license.php MIT License
  */
 namespace Cake\Localized\Validation;
 
 /**
  * Polish Localized Validation class. Handles localized validation for Poland.
  *
  */
-class PlValidation extends LocalizedValidation
+class PlValidation extends \Cake\Localized\Validation\LocalizedValidation
```

Comment:
why not only `class PlValidation extends LocalizedValidation` ?

Reply:
you can ignore, that is the same. we can clean this up with sniffers.